torch/_export/serde/serialize.py

```diff
@@ -943,12 +944,23 @@ def serialize(self, graph_module: torch.fx.GraphModule) -> GraphModule:
 
 
 class ExportedProgramSerializer:
-    def __init__(self, opset_version: Optional[Dict[str, int]] = None):
+    def __init__(
+        self,
+        opset_version: Optional[Dict[str, int]] = None,
+        convert_to_cpu: bool = False,
+    ):
+        """
+        Args:
+            opset_version: Dictionary of operator namespace to version
+            convert_to_cpu: Flag to convert all constants (ex. weights) are
+                converted to cpu device
+        """
         self.opset_version: Dict[str, int] = {}
         if opset_version:
             self.opset_version.update(opset_version)
         if "aten" not in self.opset_version:
             self.opset_version["aten"] = torch._C._get_max_operator_version()
+        self.convert_to_cpu = convert_to_cpu
 
     def serialize(self, exported_program: ep.ExportedProgram) -> SerializedArtifact:
```

Comment:
compared with a self.convert_to_cpu attibute, can we make it a default kwarg to serialize? This makes the Serializer less stateful.